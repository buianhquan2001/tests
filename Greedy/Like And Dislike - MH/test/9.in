14
43 94 94
99 59 77
73 38 59
60 81 85
58 84 73
99 61 38
46 12 90
5 75 75
59 40 29
67 39 14
95 54 24
10 33 0
83 0 88
11 65 24
