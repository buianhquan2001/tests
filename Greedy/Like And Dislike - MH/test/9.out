Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Unknown
Like
Like
Dislike
Unknown
Dislike
